src/components/RichTextEditor/EditorPositioningWrapper/ToolbarWrapper/ToolbarWrapperPositioningTop.tsx

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { ToolbarWrapperProps } from './types';
+
+export const ToolbarWrapperPositioningTop = ({ children }: ToolbarWrapperProps) => {
```

Comment:
I'm not sure if dropdown menu should close after selecting heading style in top/bottom position. Have you discussed this with Martin or Jamie?

Reply:
The functionality is the same as before the change i made. Additional to it is only fixed position of menu.
If you look into the Fondu storybook it is the same behaviour.